ports/models.py

```diff
@@ -80,3 +78,21 @@ class User(models.Model):
     os_arch = models.CharField(max_length=10)
     full_json = JSONField()
     updated_at = models.DateTimeField(auto_now=True)
+
+    def get_year(self):
+        return self.updated_at.year
+
+    def get_month(self):
+        return self.updated_at.month
+
+
+class OSDistribution(models.Model):
```

Comment:
I think I missed the discussion but why exactly is this model class needed? @mojca I think we should move that database readme from last year's repo to this one for easy access.

Reply:
We haven't discusses about it yet. I just followed the plan from my proposal- where the idea is to have separate tables for ports, os etc.

Reply:
Okay.